package.json

```diff
@@ -51,10 +51,12 @@
     "eslint": "^1.10.3",
     "eslint-config-kentcdodds": "^5.0.0",
     "eslint-plugin-mocha": "^1.1.0",
+    "execa": "^0.2.2",
```

Comment:
Tried with `npm i -S git://github.com/gtramontina/ghooks.git#fix-build` on a dummy project and works fine so far (after #37c381b).